### DISCUSSION TITLE
updateSeries data with info per point

### DISCUSSION THREAD

**Discussion by johmarjac** [apexcharts/apexcharts.js#3250] [Ideas]
Hello,

I am not sure whether this is currently possible but I dont find a way to do it as of now with the documentation. What I am trying to do is have a line chart which dynamically takes its data using ajax and then updateSeries from my API call. My API call currently only gives a date as X and a price as Y. I am creating a price history tracker. But I thought it would be useful to show more information about the entry of a price. Users on my site can add new prices. But I want them also to be able to enter, where they found the item with the price in the world. And in my chart when i hover this price entry i want a tooltip showing more info the price world location.

Its for a game where you can open up private shops in the whole world and i would like to have people the ability to put their items with price on my site but not only the price but also the location of the shop where people can find the shop to buy the item. 

so basically i need my api to not only deliver X date, Y price, it needs to send X date, Y price and INFO text which i can display in the tooltip for that data point. 

Is that possible? If not I would like this to be a feature request
